### PR TITLE
geoalchemy2 v0.15.1

### PR DESCRIPTION
**Pull request by DManowitz**
Update to v0.15.1

Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by DManowitz** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {% set name = "GeoAlchemy2" %}
-{% set version = "0.14.7" %}
+{% set version = "0.15.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/GeoAlchemy2-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geoalchemy2-{{ version }}.tar.gz
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {% set name = "GeoAlchemy2" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]:
```suggestion
{% set name = "geoalchemy2" %}
```

**Review by ocefpaf — COMMENTED** [review]
PyPI started normalizing the package names and this should fix the failures here.

**Pull request merged**